Formula/libgda.rb

```diff
@@ -21,7 +21,15 @@ class Libgda < Formula
   depends_on "libgee"
   depends_on "openssl"
   depends_on "readline"
-  depends_on "sqlite"
+
+  # this build uses the sqlite source code that comes with libgda,
+  # as opposed to using the system or brewed sqlite3, which is not supported on macOS,
```

Comment:
What happens if the system version is used? Is it too old?

Reply:
If the system version is used, the sqlite backend doesn't work at all. Due to a design decision, instead of just linking the backend plugin against the system sqlite library, it gets actually `dlopen`'ed at runtime through GLib's GModule API. Unfortunately it assumes that the sqlite library then has a `.so` suffix instead of `.dylib`, thereby being unable to load it...

The developers are aware of this, so they just don't support this on macOS.

Reply:
I will set up `inreplace` then, after `shared-mime-info`'s rev bump build finishes.

Reply:
:heart: